spec/redis_session_store_spec.rb

```diff
@@ -59,6 +59,28 @@
     end
   end
 
+  describe 'when configured with both :ttl and :expire_after' do
+    let :options do
+      {
+        key: random_string,
+        secret: random_string,
+        redis: {
+          host: 'hosty.local',
+          port: 16_379,
+          db: 2,
+          key_prefix: 'myapp:session:',
+          ttl: 60 * 120,
+          expire_after: nil
+        }
+      }
+    end
+
+    it 'assigns the :ttl option to @default_options' do
+      expect(default_options[:ttl]).to eq(60 * 120)
```

Comment:
maybe we can assign `60 * 120` to a variable and ref that here and on L72 so it's more obvious where it's coming from?